src/Defaults/baileys-version.json

```diff
@@ -1,3 +1,3 @@
 {
-	"version": [2, 2308, 7]
-}
\ No newline at end of file
+	"version": [2, 2321, 6]
```

Comment:
```suggestion
	"version": [2, 2322, 15]
```

Reply:
Could you change to latest?

Reply:
😅 I forgot I followed the wa web version

Reply:
After this merge, I will update the "proto update" workflow to update this file too